week-9/Homework/mandatory/2-exercises/1-shopping-cart.js

```diff
@@ -11,9 +11,16 @@ The output of running your code should be:
 
 class ShoppingCart {
   // Add your code here
-
+  constructor(){
+    this.items = []
+  }
+  addItem(item) {
+  this.items.push(" " + item) 
+  }
   cartContains() {
     // Use console.log() to output everything contained in your cart
+    console.log(`Your shopping cart has ${this.items.length} items: ${this.items}`)
```

Comment:
This is very close to being correct but you are not outputting everything that the question is asking you to! You are along the right lines though :)